src/include/common/types/timestamp_t.h

```diff
@@ -62,55 +62,57 @@ struct timestamp_sec_t : public timestamp_t { // NO LINT
 
 // The Timestamp class is a static class that holds helper functions for the Timestamp type.
 // timestamp/datetime uses 64 bits, high 32 bits for date and low 32 bits for time
-class Timestamp {
+class KUZU_API Timestamp {
```

Comment:
Should we move `KUZU_API` to the class level? I think we have functions not intended as APIs, such as `getTimestampConversionExceptionMsg`.